discussion.md

```diff
@@ -0,0 +1,54 @@
+#Discussion 
+
+##1. Summary of PCA paper
+The question that PCA asks is "Is there another basis which is a linear combination of the original basis that best re-expresses our dataset?" (It is quoted from the paper but I think it best expresses what the analysis does) — @Sarah Majors 
+
+PCA stands for principal component analysis is a standard tool in modern data analysis because it is a simple, non-parametric method for extracting relevant information from confusing data sets. — @Marce
+
+##2. Can anyone explain why eigen vectors play an important role in calculating PCA???
+From my understanding they are really important because they will let us find the golden matrix for change basis, based on our X matrix — @marce
+
+Eigen vectors create a orthogonal basis, which can represent our data in much better way and remove redundant dimensions — @dharmendra 
+
+##3. What is more efficient? Svd or Eigendecomposition?
+
+Both are numerically same. But how those Algorithms are implemented in framework. Some people says SVD is better due to its implementation. [Why does Andrew NG prefer SVD and not EIG of covariance matrix to PCA ?](https://stats.stackexchange.com/questions/314046/why-does-andrew-ng-prefer-to-use-svd-and-not-eig-of-covariance-matrix-to-do-pca) --@dharmendra
+
+##4. Is it necessary to do feature normalization before PCA and why ?
+Yes, I believe it is best that you normalize. Otherwise, features with considerably larger ranges could influence too much the results and you would end up with a biased final result. — @Artur Freitas Gonçalves
+
+##5. Is it necessary to have target variable for applying dimenionality reduction algorithms ???
+Target variable is not required for PCA and SVD -- @dharmendra
+
+##6. In the `Change of Basis` section, we have `PX = Y`, where P is stated to be a linear transformation applied to X. It's then stated that this equation can have many interpretations, one of which is that - `Geometrically, P is a rotation and a stretch which again transforms X into Y`. How exactly is that interpretation made? `P` is shown to be just a column vector, and if I'm not wrong, a column vector can't be a rotation and a stretch, right? So, how did they come to that conclusion?
```

Comment:
I think some `` in this question is not correctly used (also in the questions below), are you intended to use ''?
Please review this using a md viewer.